theme/less/_header.less

```diff
@@ -0,0 +1,64 @@
+header
+{
+  box-shadow: 0px 8px 15px rgba(0, 0, 0, 0.06);
+}
+
+header label select
+{
+  color: @secondary;
+  font-size: 24px;
+  border: 0;
+  cursor: pointer;
+  margin: 25px;
```

Comment:
Detail: You don't have to use margin in order to avoid border as in comparison to other items, if that was the intention. In order to move the item to the middle in the cell, you can use `.middle` class on it, so basically you might need to use only side margins. This way if the paddings on other item changes, you won't need to adjust this margin accordingly to fit them.

Same applies to the hamburger menu on mobile.

Reply:
Alternatively you can also try using [CSS variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties).

BTW, why don't we use padding here?

Reply:
Now I recall that we agreed to use padding.

Reply:
Padding is making border of `select` touch to window border. Let's discuss this on meeting.

Reply:
In the case of new version of language selector if we use padding context menu moves away from current language `button`.

Reply:
Done.